src/NServiceBus.Metrics.PerformanceCounters/PerformanceCounterUpdater.cs

```diff
@@ -28,7 +29,8 @@ public void Update(string payload)
         foreach (var timer in timers)
         {
             var performanceCounterInstance = cache.Get(new CounterInstanceName(timer.Name, context));
-            performanceCounterInstance.RawValue = timer.TotalTime;
+            
+            performanceCounterInstance.RawValue = timer.Histogram.LastValue / 1000;
```

Comment:
Maybe comment here that counter values are in seconds but measure values are in milliseconds which are reflected in the histo?